tests/test_all_fields.py

```diff
@@ -47,9 +47,7 @@
     },
     'plans': set(),
     'invoice_line_items': {
-        'tax_rates',
```

Comment:
`tax_rates` and `price` - both fields added to schema. Now it replicates in actual records also. So, removed it from the `KNOWN_MISSING_FIELDS` map.